zh/whats-new-in-v1.1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: What's New in v1.1
```

Comment:
Please update according to the comments on the google doc.

Reply:
updated